src/common_subset.rs

```diff
@@ -244,9 +255,10 @@ impl<NodeUid: Clone + Debug + Ord + Rand> CommonSubset<NodeUid> {
         self.broadcast_results.insert(proposer_id.clone(), value);
         let set_agreement_input = |agreement: &mut Agreement<NodeUid>| {
             if agreement.accepts_input() {
-                agreement.set_input(true)
+                // FIXME: Use the result.
+                agreement.input(true)
             } else {
-                Ok(FaultLog::new())
+                Ok(Default::default())
```

Comment:
`FaultLog::new()` vs `default()`, but it looks like you specifically chose to use `default` over `new` here. If that was you're intention, then this is fine with me.